docs/input_overview.rst

```diff
@@ -109,10 +109,21 @@ Output file names also contain the series number, starting at the value given by
 For the input file shown previously, the output files will start with ``vmc.s000``, for example, ``vmc.s000.scalar.dat``.
 If there were another ``<qmc>`` section in the input file, the corresponding output files would use the prefix ``vmc.s001``.
 
-``<project>`` tag accepts additional control parameters ``<parameters/>``. Batched drivers check against ``max_seconds`` and make efforts to stop the execution cleanly at the end of a block before reaching the maximum time. Classic drivers can also take the now-deprecated ``maxcpusecs`` parameter for the same effect in the per driver XML section.
+The ``<project>`` tag accepts additional control parameters (using the ``<parameters/>`` tag) that can set time limits and specify the driver version.
+
+Time limits
+~~~~~~~~~~~
+Batched drivers check against ``max_seconds`` and make efforts to stop the execution cleanly at the end of a block before reaching the maximum time. Classic drivers can also take the now-deprecated ``maxcpusecs`` parameter for the same effect in the per driver XML section.
 
 In addition, a file named ``id`` plus ``.STOP``, in this case ``vmc.STOP``, stops QMCPACK execution on the fly cleanly once being found in the working directory.
 
+
+Driver version
+~~~~~~~~~~~~~~
+The ``driver_version`` parameter selects between the new performance-portable batched drivers and the previous drivers (now referred to as the 'legacy drivers').
+The values for this parameter are ``legacy`` or ``batch`` (alternately, ``batched``).
```

Comment:
Do we actually support both batch and batched?

Reply:
https://github.com/QMCPACK/qmcpack/blob/e99b6156f8df68c2501ed16388ecde0668e22530/src/Utilities/ProjectData.h#L54